src/LogViewer/index.js

```diff
@@ -178,12 +182,29 @@ export default class LogViewer extends React.Component {
 
   renderChunks() {
     return (
-      <LazyList list={this.state.chunkHeights} buffer={2}>
+      <LazyList
+        list={this.state.chunkHeights}
+        buffer={2}>
         {(heights, shouldLoad) => heights.map((height, index) => this.renderChunk(height, index, shouldLoad))}
       </LazyList>
     );
   }
 
+  jumpToLine(lineNumber) {
+    const { minLineHeight, offset } = this.state;
```

Comment:
I don't see this method being called outside of `jumpToHighlight`. I think we need to fetch the `lineNumber` for jumping from a new query string/state entry, and then handle jumping once it's loaded, similar to how it's being done in `jumpToHighlight`.